### DISCUSSION TITLE
How to render python code blocks???

### DISCUSSION THREAD

**Discussion by athrvvvv** [squidfunk/mkdocs-material#4250] [Q&A] [answered]
Hey there, iam writting an documentation with help of mkdocs and readthedocs.
I have got into an problem

Iam currently using **python 3.9** on **windows**

When I try to render python code block via `pymdownx.superfences` extension it **converts whole code block into single-single inline code block**

---

### I have this in my `mkdocs.yml`
![image](https://user-images.githubusercontent.com/98272709/184622615-c236f0b4-0743-4688-916d-892ad62f1230.png)

---

### **I tried**
![image](https://user-images.githubusercontent.com/98272709/184622706-5ce1d41f-0e9c-4d9b-9b29-cfaf5afec7f3.png)

### **This gives output as**

![image](https://user-images.githubusercontent.com/98272709/184622801-d103e84b-786d-4377-ab99-579b6ba544bc.png)

---

### **I also tried**

![image](https://user-images.githubusercontent.com/98272709/184622918-08bebd81-e53f-46e8-8b8f-fc8de8e42cc0.png)

**but problem remains the same**

Please suggest me some solutions. waiting for answer 🙏

**Comment by squidfunk** [comment, marked as answer]
Duplicate of #4251. Please stop double-posting discussions __and__ issues.
